pcap-linux.c

```diff
@@ -303,7 +303,7 @@ typedef int		socklen_t;
  */
 struct pcap_linux {
 	u_int	packets_read;	/* count of packets read with recvfrom() */
-	long	proc_dropped;	/* packets reported dropped by /proc/net/dev */
+	long	sysfs_dropped;	/* packets reported dropped by /sys/class/net */
```

Comment:
Perhaps that should be "...reported dropped in /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.